lib-esm/ActionList/Description.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import Box from '../Box.js';
-import '../sx.js';
+import '@styled-system/css';
```

Comment:
this feels like a strange replacement?

also why was `import '../sx.js'` was imported without any import specifiers??